doc/model/train-fitting-tensor.md

```diff
@@ -81,9 +102,48 @@ The JSON of `polar` type should be provided like
 -   `sel_type` is a list specifying which type of atoms have the quantity you want to fit. For example, in the water system, `sel_type` is `[0]` since `0` represents atom `O`. If left unset, all types of atoms will be fitted.
 -   The rest arguments have the same meaning as they do in `ener` mode.
 
+:::
+
+:::{tab-item} PyTorch {{ pytorch_icon }}
+
+The JSON of `dipole` type should be provided like
+```json
+	"atom_exclude_types": [
+      1
+    ],
+	"fitting_net" : {
+		"type": "dipole",

```

Comment:
https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/train/train-input.html#model-standard-fitting-net-dipole

We may remove `
(Supported Backend: TensorFlow)`

Reply:
Also for polar